cpp/src/gandiva/gdv_function_stubs.h

```diff
@@ -39,11 +39,14 @@ uint8_t* gdv_fn_context_arena_malloc(int64_t context_ptr, int32_t data_len);
 
 void gdv_fn_context_arena_reset(int64_t context_ptr);
 
-bool in_expr_lookup_int32(int64_t ptr, int32_t value, bool in_validity);
+bool gdv_fn_in_expr_lookup_int32(int64_t ptr, int32_t value, bool in_validity);
```

Comment:
why are these changes in this patch?

Reply:
I just removed these corrections.
I noticed that these functions definitions on header are incorrect defined compared to their implementation on the gdv_functions_stubs.cc, so I just corrected them here. But it seems fine to put this on a separate PR

Reply:
oh if these are corrections no issues then. I thought these were some unintended changes included.